bin/node/src/commands/mod.rs

```diff
@@ -95,4 +100,12 @@ pub struct BlockProducerConfig {
     /// Maximum number of batches per block.
     #[arg(long = "max-batches-per-block", env = ENV_MAX_BATCHES_PER_BLOCK, value_name = "NUM", default_value_t = DEFAULT_MAX_BATCHES_PER_BLOCK)]
     pub max_batches_per_block: usize,
+
+    #[arg(
+        long = "mempool.tx-capacity",
+        default_value_t = miden_node_block_producer::DEFAULT_MEMPOOL_TX_CAPACITY,
+        env = ENV_MEMPOOL_TX_CAPACITY,
+        value_name = "NUM"
+    )]
+    mempool_tx_capacity: NonZeroUsize,
```

Comment:
An alternative configuration would be to do it as a maximum time capacity.

i.e. `60s` and then the actual amount is derived from the configured block rate, and block/batch size.

I'll leave it as is for now.